packages/design-tokens/src/tokens/functional/components/button/colors.js

```diff
@@ -63,5 +63,127 @@ module.exports = {
         },
       },
     },
+    button: {
```

Comment:
Noticed this is lowercase here, so assuming this related to the the new naming convention being proposed where components / patterns aren't PascalCase?

Also, should we remove all the lines above this as those `--brand-Button-etc` variables are probably redundant now?

Reply:
Yes, exactly.

Yeah I mentioned deprecation in the PR description as I'm not sure.. if I delete them and they are used outside this repo, breaking change. What do you think?

Reply:
I'd go ahead and remove them. We can call out in a changeset which tokens have been replaced. You should be able to just take the diff from the PR comment and paste it into the changeset. WDYT?